src/planscape/config/conditions.json

```diff
@@ -191,6 +191,7 @@
                     "pillar_name": "air_quality",
                     "filepath": "data/sierra_nevada/air_quality",
                     "display_name": "Air Quality",
+                    "display": "False",
```

Comment:
can we represent this as a bool instead of a string?

Reply:
Done!